torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -135,8 +117,11 @@ Reducer::Reducer(
 
         // Hook to execute after the gradient accumulator has executed.
         hooks_.emplace_back(
-            grad_accumulator->add_post_hook(torch::make_unique<LambdaPostHook>(
-                [=] { this->autograd_hook(index); })),
+            grad_accumulator->add_post_hook(
+                torch::make_unique<torch::autograd::utils::LambdaPostHook>(
+                    [=](const torch::autograd::variable_list& /* unused */) {
```

Comment:
what's this parameter for given that it's not used?

Reply:
Its not used in this case, but you can check the internal changed in D20958327 where we use the inputs parameter. That is why we had to change the signature as well.

Reply:
I see, this make sense.